### DISCUSSION TITLE
mark POI categories as favorites and move to the top

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#11] [Q&A] [answered]
It is not very comfortable to search through tons of POI categories for every search. Can you please make the categories that are mostly or preferably used markable as favorite and move these to the top of the list?

**Comment by devemux86** [comment]
Thanks for the suggestion Tom!

Currently the plan is to integrate the offline POI into the global Search (with favorites, etc.).
So we should have 1 place to search for everything. The results will naturally be grouped.

View the POI categories is something different and indeed needs improvements.
Another idea is to _filter the categories_ when writing in their window's name field.

**Comment by devemux86** [comment, marked as answer]
Let's continue the discussion in the main topic of Cruiser search at https://github.com/devemux86/cruiser/discussions/74.
